src/site/outline/outline.rst

```diff
@@ -110,63 +91,58 @@ Below is a list of topics in no particular order that may or may not be covered
 * Sorting Algorithms
 
 
-
 Student Evaluation (Tentative Dates)
 ====================================
 
 +------------------------+------------+---------------------+
 |                        | Worth      | Due                 |
 +========================+============+=====================+
-| Assignment 1           | 10%        | Early February-ish  |
+| Assignment 1           | 5%         | Late January-ish    |
 +------------------------+------------+---------------------+
-| Assignment 2           | 10%        | Late February-ish   |
+| Assignment 2           | 5%         | Mid February-ish    |
 +------------------------+------------+---------------------+
-| Assignment 3           | 10%        | Mid March-ish       |
+| Assignment 3           | 5%         | Mid March-ish       |
 +------------------------+------------+---------------------+
-| Assignment 4           | 10%        | Early April-ish     |
+| Assignment 4           | 5%         | Early April-ish     |
 +------------------------+------------+---------------------+
-| Labs                   | 10%        | Throughout          |
+| Test 1                 | 20%        | Late January-ish    |
 +------------------------+------------+---------------------+
-| Midterm Exam           | 20%        | Early March-ish     |
+| Test 2                 | 20%        | Late February-ish   |
 +------------------------+------------+---------------------+
-| Final Exam             | 30%        | TBA                 |
+| Final Exam             | 40%        | TBA                 |
 +------------------------+------------+---------------------+
 
 
+Assignments
+===========
 
-Assignment Submission
-=====================
-
-Instructions for the submission of assignments will be posted on the course website. It is each student's
-responsibility to read and follow the instructions. Failure to follow the submission instructions may result in the
-assignment receiving a mark of zero.
-
-You will be required to submit each programming assignment electronically. Details will be given in the assignment
-descriptions.
-
-We reserve the right to use similarity detection software to detect possible cheating cases.
 
+Submission
+----------
 
+Instructions for the submission of assignments will be posted on the course website. It is each student's responsibility
+to read and follow the instructions. Failure to follow the submission instructions may result in the assignment
+receiving a mark of zero.
 
-Assignment Due Dates
-====================
+You will be required to submit each programming assignment electronically. Details will be given in the assignment
+descriptions. We reserve the right to use similarity detection software to detect possible cheating cases.
```

Comment:
_We reserve the right_? Just say you _will check for cheating_ using blah blah.

Reply:
I'll check if that is official wording.